Example/App.xaml.cs

```diff
@@ -18,10 +25,64 @@ public App ()
 
 		protected override void OnStart ()
 		{
-            
-
+            validateCert();
 		}
 
+        public async void validateCert()
+        {
+            try 
+            {
+                string url;
+
+                MobileCoreConfiguration configuration = MobileCore.Instance.Configuration;
+
+                IEnumerator<ServiceConfiguration> serviceConfigurations = configuration.ServiceConfigurations.GetEnumerator();
+
+                if (serviceConfigurations.MoveNext())
+                {
+                    url = serviceConfigurations.Current.Url;
```

Comment:
so it checks only first URL in the configurations? Is this desired behaviour?

Reply:
@wojta This is what we agreed to do for the other showcase apps for now